src/odemis/acq/test/stream_test.py

```diff
@@ -4312,6 +4346,72 @@ def test_ebic_acq(self):
         numpy.testing.assert_allclose(ebic_md[model.MD_PIXEL_SIZE], exp_pxs)
         self.assertEqual(ebic_md[model.MD_USER_TINT], (255, 0, 0))  # from .tint
 
+    def test_acq_cl_se_ebic(self):
+        """
+        Test acquisition for SE+CL+EBIC acquisition simultaneously (aka "folded stream")
+        """
+        # create axes
+        axes = {"filter": ("band", self.filter)}
+
+        # Create the stream
+        sems = stream.SEMStream("test sem", self.sed, self.sed.data, self.ebeam,
+                                emtvas={"dwellTime", "scale", "magnification", "pixelSize"})
+        mcs = stream.CLSettingsStream("testCL",
+                                      self.cl, self.cl.data, self.ebeam,
+                                      axis_map=axes,
+                                      emtvas={"dwellTime", })
+        ebics = stream.IndependentEBICStream("test ebic", self.ebic, self.ebic.data,
+                                             self.ebeam, self.sed.data,
+                                             emtvas={"dwellTime"})
+
+        sms = stream.SEMMDStream("test sem-md", [sems, mcs, ebics])
+
+        # Now, proper acquisition
+        mcs.roi.value = (0, 0.2, 0.3, 0.6)
+        mcs.tint.value = (255, 0, 0)  # Red colour
+        ebics.roi.value = (0, 0.2, 0.3, 0.6)
+        ebics.tint.value = (0, 255, 0)  # Green colour
```

Comment:
As the above settings of the roi area are the same between two different detectors, it is expected to yield same metadata. Would it be useful to test if the roi settings change slightly to some extent, it can still provide same metadata or not?

Reply:
If the `roi` values are not identical, simultaneous acquisition will fail. So we cannot test when the values are different. 
In other test cases we do test that the roi settings are actually enforced and compare them with the final metadata. So it's should be covered already.